packages/@webex/internal-plugin-metrics/src/batcher.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable require-jsdoc */
```

Comment:
why now its throwing error and not before ?

Reply:
I have fixed this too.